components/Button/RandomButton.styled.js

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { Merriweather_Sans } from "next/font/google";
```

Comment:
I don't know why you import and declare the font-family here since it is defined to the whole `body` in the `styles.js`. And it seems as you use it everywhere.

Reply:
Since the font was not sent here for some reason. Can you tell me why?

Reply:
Ok I have found it. Is now in Global and only in one place :)